roles/foreman_installer/tasks/trust_github.yml

```diff
@@ -0,0 +1,35 @@
+---
+- name: 'Set GitHub RSA Public Key'
+  set_fact:
+    github_rsa_public_key: 'ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAq2A7hRGmdnm9tUDbO9IDSwBK6TbQa+PXYPCPy6rbTrTtw7PHkccKrpp0yVhp5HdEIcKr6pLlVDBfOLX9QUsyCOV0wzfjIJNlGEYsdlLJizHhbn2mUjvSAHQqZETYP81eFzLQNnPHt4EVVUh7VfDESU84KezmD5QlWpXLmvU31/yMf+Se8xhHTvKSCZIFImWwoG6mbUoWf9nzpIoaSjB+weqqUUmpaaasXVal72J+UX2B+2RPW3RcT0eOzQgqlJL3RKrTJvdsjE3JEAvGq3lGHSZXy28G3skua2SmVi/w4yCE6gbODqnTWlg7+wC604ydGXA8VJiS5ap43JXiUFFAaQ=='
+
+- name: "install git"
+  package:
+    name: 'git'
+    state: 'present'
+
+- name: 'Pull from github over https'
+  git_config:
+    name: 'url."git@github.com:".pushInsteadOf'
+    value: 'https://github.com/'
+    scope: system
```

Comment:
I don't like the current implementation, mostly because this 'Pull from github over https' is for an https configuration, not an ssh configuration; however, I would use the https remote now that I know about this config in Git. Because of this paired with SSH agent forwarding, we no longer need to have public RSA keys on the box itself. I'm going to change this PR accordingly and also create a new foreman_devel role to contain this type of logic, as suggested above.